docs/tutorials/fundamentals/quick_start.md

```diff
@@ -0,0 +1,204 @@
+
+Additional input file formats may be supported [by plugins](https://www.napari-hub.org/). 
+Try [napari-aicsimageio](https://www.napari-hub.org/plugins/napari-aicsimageio) if you have czi, lif, or nd2 files.
+
+Once you have the proper plugin installed, use File > Open Files(s)... and select the image file, or simply drag and drop the image into napari. 
+
+For demo purpose, we will use a sample image that comes with napari.
+
+(1) Open napari IPython console<br>
+<img src="./images/IPython.png" alt="IPython console" width="30%"/>
+
+(2) Type
+    
+```python
+from skimage import data
+viewer.add_image(data.cell(), name='cell')
+```
+
+```{code-cell} ipython3
+:tags: [remove-input]
+
+import napari
+from napari.utils import nbscreenshot
+
+viewer = napari.Viewer()
+
+from skimage import data
+viewer.add_image(data.cell(), name='cell')
+nbscreenshot(viewer)
+```
+
+### Image display adjustment
+
+The layer controls panel at the upper left of the viewer allows you to adjust contrast, colormap, and other layer properties. These settings affect the visualization, but do not affect the underlying data.
+
+To change the image display through the [API](api/index), in IPython console, type
+
+```python
+viewer.layers['cell'].colormap = "yellow"
+```
+
+```{code-cell} ipython3
+:tags: [remove-input]
+
+viewer.layers['cell'].colormap = "yellow"
+nbscreenshot(viewer)
+```
+
+### Manually label the cell
+
+To measure the area of the cell, we can use a labels layer and manually "paint" the cell.
+The labels layer allows you to record the segmentation result by assigning background = 0, and assigning each object with an integer.
+
+1. Add a new labels layer 
+1. Click on "paint"
+1. Circle the cell
+1. Use "fill" bucket to fill it.
+
+![manual_label](./images/manual_label.gif)
+
+Several plugins can perform automatic segmentation that takes image layer as input and generates labels layer as output.
+
+Try [cellpose-napari](https://www.napari-hub.org/plugins/cellpose-napari) if you have cell images.
+
++++
+
+### Get the cell area measurement
+
+To analyze labels layer, we can use [scikit-image](https://scikit-image.org/), a popular Python library that comes with napari installtion. [skimage.measure.regionprops](https://scikit-image.org/docs/dev/api/skimage.measure.html#skimage.measure.regionprops) provides a good set of features that can be extracted from labels, including area measurement.
+
+In IPython console, type
+
+```python
+from skimage.measure import regionprops
+props = regionprops(viewer.layers['Labels'].data)
+print("the cell area is: ", props[0].area)
+```
+
+Alternatively, try [this plugin](https://www.napari-hub.org/plugins/napari-skimage-regionprops) to have the result in a table form.
+
+**Note:** the area reported by regionprops is the number of pixels. Check pixel size and convert the reported number to physical units.
+
+```{code-cell} ipython3
+:tags: [remove-input]
```

Comment:
```suggestion
:tags: [remove-cell]
```

We can still see a little `1` being printed if we only remove the input, so we should remove the whole cell here (output included).